packages/flutter_tools/test/integration.shard/hot_reload_test.dart

```diff
@@ -43,7 +44,13 @@ void main() {
 
     test('newly added code executes during hot reload', () async {
       await _flutter.run();
+      if (Platform.isWindows) {
```

Comment:
My current guess is that _slight_ changes in the recorded timestamp have tripped up the windows filesystem. Since the files are modified synchronously in the test environment, perhaps the last recorded timestamp is identical to the first compiled time?

I tried a few different approaches like keeping the datetime initialization before the call to compile and only rolling it back afterwards, but that wasn't sufficient.

Reply:
Ahh fails on CI, passes locally ... more time?

Reply:
I moved the creation of the lastCompiledTime all the way back to the start of the method... that plus the awaits should help....

Reply:
Removing the awaits, that was all that we needed.

Reply:
The issue was likely that removing `DateTime.now()` to after the compilation placed it closed enough to the next invalidation that the with the windows file-system the timestamp granularity was not sufficient.